src/loaders/index.js

```diff
@@ -7,7 +7,7 @@ const loadHttpServer = require("./server");
 
 const indexRouter = require("../routes/index");
 
-const initLoaders = async (app) => {
+const initLoaders = (app) => {
   logger.info("app start");
 
   loadDatabase();
```

Comment:
```suggestion
 await loadDatabase();
```
loadDatabase 를 프로미스로 해서 db에 연결 될때만 서버가 실행 되게끔 해두 될 것 같아요
이건 이야기를 나눠봐야 될 것 같아요

Reply:
확인했습니다! 논의한 방향이 더 좋은 것 같아요 😄